lib/web_console/middleware.rb

```diff
@@ -18,8 +18,9 @@ class Middleware
 
     UNACCEPTABLE_REQUEST_MESSAGE = "A supported version is expected in the Accept header."
 
-    cattr_accessor :whiny_requests
-    @@whiny_requests = true
+    cattr_accessor :verbose_logging, :acceptable_content_types
+    @@verbose_logging = true
+    @@acceptable_content_types = [Mime::HTML]
```

Comment:
I'm not longer using the `acceptable_content_types` config setting because I guessed the same thing you're saying: it was an undocumented option to handle buggy cases.

I forgot to remove this line, so let's :scissors: